### ISSUE TITLE
Electric Drummer Update

### ISSUE THREAD

**Issue by Kelly-NAproducR** [scripting/drummerSupport#172]
Dave,
Is there an Electric Drummer update for v2.0.16 ?
Will there be Bookmarklet support in E/D?

Thank you!

Kelly

**Comment by scripting** [comment]
We could do a new release of Electric Drummer. The version I'm using is v2.0.19. I never hear from E/D users, so it's hard to justify doing releases. But now that I have heard from one, I'll add it to the list.

Bookmarklets can't work with desktop apps, so I'm afraid that feature is strictly for the web version of Drummer.

**Comment by mistersugar** [comment]
I have started to use E/D again to write to my blog and to keep my daily work notes.

**Comment by scripting** [comment]
@mistersugar -- do you also use micro.blog?

i've started writing posts there using E/D.

**Comment by mistersugar** [comment]
@scripting yes, I do use Drummer to post to my Micro.blog account and it is smooth (I'm @antonzuiker there, blog [here](https://smol.zuiker.com)). I'm also following your ongoing conversation with Manton and thankful for the interop.

**Comment by scripting** [comment]
@mistersugar -- i want it to work better. i think it should do everything that old school does when rendering a blog. the code he needs is open source, and i would help him. i want to start building interop with him, not just having nice conversations about it. prime the pump.

**Comment by scripting** [comment]
@mistersugar -- i just followed you. how can i find other drummer users who are on micro.blog?

**Comment by Kelly-NAproducR** [comment]
@scripting -- I understand. It's not urgent. Hopefully others will speak up about an update. My OS update removed E/D, so I thought I'd ask before reinstalling E/D. TY!  

> We could do a new release of Electric Drummer. The version I'm using is v2.0.19. I never hear from E/D users, so it's hard to justify doing releases. But now that I have heard from one, I'll add it to the list.
